utilities/tasks/addSearchesFieldsToWobjects/addSearchesFieldsToWobjects.js

```diff
@@ -0,0 +1,47 @@
+const _ = require('lodash');
+const { parseAddress } = require('utilities/helpers/updateSpecificFieldsHelper');
+const { FIELDS_NAMES } = require('constants/wobjectsData');
+const { WObject } = require('database').models;
+
+module.exports = async (fieldName) => {
+  if (fieldName === 'author_permlink') return getAuthorPermlinks();
+  const wobjects = await WObject.find({ 'fields.name': fieldName }, { author_permlink: 1, fields: 1 }).lean();
+  for (const wobj of wobjects) {
+    const { newFields, err } = await getFieldsData(wobj.fields, fieldName);
+    if (err) {
+      console.error(`Failed to parse the address for the object: ${wobj.author_permlink} with error:`, err);
+      continue;
+    }
+    await WObject.updateOne({ _id: wobj._id }, { $set: { [`search.${fieldName}`]: newFields } });
+  }
+};
+
+const getFieldsData = async (fields, fieldNameForParse) => {
+  const newFields = [];
```

Comment:
.chain().filter().map()
create object to store callbacks for map instead switch case